lib/IDE/ExprContextAnalysis.cpp

```diff
@@ -289,7 +290,9 @@ static void collectPossibleCalleesByQualifiedLookup(
     Type declaredMemberType = VD->getInterfaceType();
     if (VD->getDeclContext()->isTypeContext()) {
       if (isa<FuncDecl>(VD)) {
-        if (!isOnMetaType)
+        if (!isOnMetaType && VD->isStatic())
+          continue;
+        if (isOnMetaType == VD->isStatic())
```

Comment:
I thought I understood this for the case `isOnMetaType  == false && VD->isStatic() == false`, but why do we do this also when `isOnMetaType == true && VD->isStatic() == true`?

Reply:
Static method also has curried interface type. e.g.
```swift
class C {
  func foo(_: Int) -> Void {}
}
```
The interface type of `C.foo(_:)` is `(C.Type) -> (Int) -> Void`.

Reply:
```
class C {
  static func foo(arg1: String, arg2: String) -> C {
    return C()
  }
}
func test() {
  let _: C = C.foo(arg1: "test", #^HERE^#)
}
```
In Xcode 10.2.1, this completion doesn't suggest `arg2:` because of this bug.